### PR TITLE
Domd and mmc integration

### PR DESCRIPTION
**Pull request by oleksiimoisieiev**

### CODE REVIEW

File: boards/rcar_spider.conf
Comment (al1img) [inline boards/rcar_spider.conf:35]: New line at the end

File: .clang-format
Comment (al1img) [inline .clang-format:10]: Remove empty line

File: west.yml
Comment (xakep-amatop) [inline west.yml:33]: (question) maybe is it better to wait until I merge related changes to aos-wip?

Comment (oleksiimoisieiev) [reply west.yml:33]: I know. But we have no time ((

File: src/main.cpp
Comment (xakep-amatop) [inline src/main.cpp:18]: (note) you can use `CONFIG_SOC_SERIES_RCAR_GEN4` and `CONFIG_SOC_SERIES_RCAR_GEN3` instead
if we want to add other boards in the future

File: src/domd_cfg_spider.c
Comment (xakep-amatop) [inline src/domd_cfg_spider.c:14]: (note) I'm not sure that this is correct, at far as I know we should delete xen pass through property for MMC and I thought that domd doesn't able to work with this device after it

Comment (oleksiimoisieiev) [reply src/domd_cfg_spider.c]: dropped mmc node from the config

File: src/domd_cfg_gen3.c
Comment (oleksiimoisieiev) [reply src/domd_cfg_gen3.c:25]: +

File: boards/rcar_h3ulcb_ca57.conf
Comment (xakep-amatop) [inline boards/rcar_h3ulcb_ca57.conf:32]: (note) there isn't new line at the end of the file

**Review by xakep-amatop — APPROVED** [review]
review has been finished, I haven't found major issues
pls, run checkpath and fix all code style issues

`Acked-by: Mykola Kvach <mykola_kvach@epam.com>`

File: boards/rcar_salvator_xs_m3_2x4g.conf
Comment (oleksiimoisieiev) [reply boards/rcar_salvator_xs_m3_2x4g.conf]: +

**Pull request merged**
